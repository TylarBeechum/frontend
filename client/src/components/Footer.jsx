import React from 'react';
import styled from 'styled-components';

const Footer = () => {
  return (
    <FooterContainer>
      <span>About</span>
      <span>stampd2019</span>
      <span>Contact Us</span>
    </FooterContainer>
  );
};
const FooterContainer = styled.div`
  display: flex;
  justify-content: space-evenly;
  border-top: 1px solid #333;
  position: relative;
  top: 75vh;
`;

export default Footer;
