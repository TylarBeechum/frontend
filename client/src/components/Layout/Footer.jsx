import React from 'react';
import styled from 'styled-components';

const Footer = () => {
  return (
    <FooterContainer>
      <span>Â© Stampd 2019</span>
    </FooterContainer>
  );
};
const FooterContainer = styled.div`
  text-align: center;
  position: fixed;
  bottom: 0;
  width: 100%;
  color: #333333;
  background: #f8f8f8;
  padding-bottom: 10px;
`;

export default Footer;

// ${props => props.theme.global.colors['neutral-3']}
// ${props => props.theme.global.colors['accent-3']}
