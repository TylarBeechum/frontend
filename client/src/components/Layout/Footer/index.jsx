import React from 'react';
import styled from 'styled-components';

const Footer = props => {
  return (
    <FooterContainer>
      <span>stampd2019</span>
    </FooterContainer>
  );
};
const FooterContainer = styled.div`
  text-align: center;
  border-top: 1px solid #333;
  position: fixed;
  bottom: 0;
  width: 100%;
  color: white;
  background-color: #3aecfc;
  border-top: 2px solid #82fdff;
`;

export default Footer;

// ${props => props.theme.global.colors['neutral-3']}
//${props => props.theme.global.colors['accent-3']}
