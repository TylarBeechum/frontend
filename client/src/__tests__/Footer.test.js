/* eslint-disable */
import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import {
  renderWithRouterAndProviders,
  rendererWithRouterAndProviders
} from '../__testHelpers__';

import Footer from '../components/Layout/Footer';

describe('<Footer />', () => {
  describe('should render the Footer component successfully without crashing', () => {
    it('tests that the component does not crash when it is rendered', () => {
      const helpers = renderWithRouterAndProviders(<Footer />);
    });
  });

  describe('should match the previous snapshot of the component', () => {
    it('tests that the component has not changed and matched the previous snapshot', () => {
      const tree = rendererWithRouterAndProviders(<Footer />);
      expect(tree.toJSON()).toMatchSnapshot();
    });
  });

  describe('should render the copyright string successfully', () => {
    it('tests that the copyright string is rendered in the footer', () => {
      const { getByText } = renderWithRouterAndProviders(<Footer />);
      const copyright = getByText(/Â© Stampd 2019/i);
      expect(copyright).toBeDefined();
    });
  });
});
