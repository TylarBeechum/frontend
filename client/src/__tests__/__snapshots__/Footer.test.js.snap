// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should match the previous snapshot of the component tests that the component has not changed and matched the previous snapshot 1`] = `
<div
  className="StyledGrommet-sc-19lkkz7-0 iDwiuh"
>
  <div
    className="sc-EHOje jibMrV"
  >
    <p>
      Â© Stampd 2019
    </p>
  </div>
</div>
`;
